### PR TITLE
all TFs passed to reco

### PR DESCRIPTION
**Pull request by AllaMaevskaya**
- rename Digits branches in file with digits generated from raw;
- fix bug and allow reconstruction of all TFs

### CODE REVIEW

**Review by jotwinow — APPROVED** [review]

**Comment by AllaMaevskaya** [comment]
Hello
what is wrong here? Can it be merged?

**Review by mslupeck — APPROVED** [review]
To me this looks OK

File: Detectors/FIT/FT0/workflow/src/ReconstructionSpec.cxx [hunk 1]
```diff
@@ -65,7 +64,6 @@ void ReconstructionDPL::run(ProcessingContext& pc)
   pc.outputs().snapshot(Output{mOrigin, "RECCHDATA", 0, Lifetime::Timeframe}, mRecChData);
 
   mFinished = true;
-  pc.services().get<ControlService>().readyToQuit(QuitRequest::Me);
```
Comment (sawenzel) [inline Detectors/FIT/FT0/workflow/src/ReconstructionSpec.cxx:68]: why do you take this away?

File: Detectors/FIT/FT0/workflow/src/ReconstructionSpec.cxx [hunk 2]
```diff
@@ -33,9 +33,6 @@ void ReconstructionDPL::init(InitContext& ic)
 
 void ReconstructionDPL::run(ProcessingContext& pc)
 {
-  if (mFinished) {
```
Comment (sawenzel) [inline Detectors/FIT/FT0/workflow/src/ReconstructionSpec.cxx:36]: why do you take this away?

File: Detectors/FIT/FT0/workflow/src/ReconstructionSpec.cxx (hunk 1 above)
Comment (shahor02) [reply Detectors/FIT/FT0/workflow/src/ReconstructionSpec.cxx:68]: Actually, both the ``pc.services().get<ControlService>().readyToQuit(QuitRequest::Me)`` and ``mFinished = true;`` (which is never used elsewhere) should go away: the reconstruction should not decide that it finished its job, this is done by the upstream data supplier, which will ``ctx.services().get<o2f::ControlService>().endOfStream()`` once the data is over.

Comment (sawenzel) [reply Detectors/FIT/FT0/workflow/src/ReconstructionSpec.cxx:68]: Ok. Thanks for the explanation.

**Review by shahor02 — APPROVED** [review]
@AllaMaevskaya merging, but at next PR please remove unused mFinished.

**Pull request merged**